app/src/main/kotlin/com/wire/android/ui/home/conversations/messages/item/RegularMessageItemLeading.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Wire
+ * Copyright (C) 2024 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+package com.wire.android.ui.home.conversations.messages.item
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import com.wire.android.model.Clickable
+import com.wire.android.model.UserAvatarData
+import com.wire.android.ui.common.UserProfileAvatar
+import com.wire.android.ui.home.conversations.model.MessageHeader
+
+@Composable
+fun RegularMessageItemLeading(
```

Comment:
question: what's the point/advantage of splitting it? Now we have 4 different composables instead of 2, for me having them encapsulated was better - we had only 2 types, previews was containing the whole message UI with avatar instead of just content, these "leading" elements should be integral parts of the message anyway and shouldn't be used separately, so I preferred to have it combined unless there's something other important that I'm missing?